src/keyListener.ts

```diff
@@ -39,13 +40,14 @@ export default class KeyListener {
     onKeyPressExecuteEvent(str: string, key: any): void {
         const foundRelatedEvent = this.events.find(
             (e) =>
+                this.validKeys.includes(key.name) &&
                 (Array.isArray(e.keyName) ? e.keyName.includes(key.name) : e.keyName === key.name) &&
                 (e.ctrl ? key.ctrl : true) &&
                 (e.shift ? key.shift : true),
         );
 
         if (foundRelatedEvent) {
             foundRelatedEvent.handler(key.name, key.ctrl, key.shift);
-        }
+        } else console.log('Invalid command.\n');
```

Comment:
Here you should wrap the log instruction inside blocks, like this:

```suggestion
        } else {
            console.log('Invalid command.\n');
        }
```

Reply:
Agreed! I applied this suggestion to a second commit.